tweets random reminders on Twitter!

ShruAgarwal opened the pull request:
<!-- If your PR fixes an open issue, use `Closes #101` to link your PR with the issue. #101 stands for the issue number you are fixing -->

## 🛠️ Fixes Issue

Closes  #6  

## 👨‍💻 Changes proposed

I have made changes to `main.py` file where it selects a random line from `reminders.txt`  and sends it on Twitter. 
I've already tried and tested it so that it tweets after every 4 hours! 

## ✔️ Check List (Check all the applicable boxes) <!-- Follow the below conventions to check the box -->

<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
<!--
[x] - Correct; marked as done
[ ] - Not correct; marked as **not** done
-->

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

I'm open to feedback for this PR. 😃

## 📷 Screenshots

ykdojo commented on main.py:
```diff
@@ -1,9 +1,37 @@
+consumer_key = ""
+consumer_secret = ""
+access_token = ""
+access_token_secret = ""
```
Thank you so much! Any chance you could put this in a separate .env file so it's easier to edit this file?

ShruAgarwal replied:
I have added a new file `load_env.py` which has a function for loading all the credentials from an `.env` file. 
Make sure to add the *secret keys* in a safe place before hosting the bot. 

**P.S.: If you're planning to host this bot on Heroku, then you can add all the secret keys within Settings > Config Vars**

ykdojo commented on main.py:
```diff
@@ -1,9 +1,37 @@
+     api.update_status(status=remind)
+     # sleeps for 4 hours
+     sleep(14400)  
+   else:
```
I'm not sure what this else clause is for?

ShruAgarwal replied:
Sorry, I wrote that by mistake..😅

ShruAgarwal commented:
Hey @ykdojo 👋
I have made changes in the `main.py` file as suggested. 
Added a new file `load_env.py` for loading all your secret keys! Can't commit .env file cause it's risky! 
Lastly, added `requirements.txt` file for keeping records of all packages.

Let me know your thoughts! 😃

ykdojo commented:
That's perfect. Thank you so much!!

ShruAgarwal commented:
Had fun contributing in this project!

Always ready to help 🤗
